Core/Utilities/RabbitMQAdmin.py

```diff
@@ -0,0 +1,146 @@
+"""RabbitMQAdmin module serves for the management of the internal RabbitMQ
+   users database. It uses rabbitmqctl command. Only the user with the right
+   permissions can execute those commands.
+"""
+import re
+from DIRAC import S_OK
+from DIRAC.Core.Utilities.DErrno import DError
+import errno
+from DIRAC.Core.Utilities import Subprocess
+
+def executeRabbitmqctl(arg, *argv):
+  """Executes RabbitMQ administration command.
+    It uses rabbitmqctl command line interface.
+    For every command the -q argument ("quit mode")
+    is used, since in some cases the output must be processed,
+    so we don't want any additional informations printed.
+  Args:
+    arg(str): command recognized by the rabbitmqctl.
+    argv(list): optional list of string parameters.
+  Returns:
+    S_OK:
+    Derror:
+
+  """
+  command =['sudo','/usr/sbin/rabbitmqctl','-q', arg] + list(argv)
+  timeOut = 30
+  result = Subprocess.systemCall(timeout = timeOut, cmdSeq = command)
+  if result['OK']:
+    returncode, cmd_out, cmd_err = result['Value']
+  else:
+    return DError(errno.EPERM, "%r failed, status code: %s stdout: %r stderr: %r" %
+                                (command, returncode, cmd_out, cmd_err) )
+  if returncode != 0:
+    # No idea what errno code should be used here.
+    # Maybe we should define some specific for rabbitmqctl
```

Comment:
Maybe rabbit mq returns standard error number ? In which case you could just forward it